src/context/themeContext.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
context folder is not good idea.
You can specify theme folder where you can define:
theme.ts with all info
ThemeProvider.tsx
ThemeConsumer.tsx or smt like that (HOC to pass props in components)